app/routes/icon-catalog/index.ts

```diff
@@ -30,9 +30,19 @@ export default class IconCatalogIndexRoute extends Route {
     if (params.label) {
       query['filter[label]'] = params.label;
     }
+    const icons = await this.store.query('icon', query);
+
+    const sortedIcons = icons.toArray().sort((a, b) => {
+      const labelA = (a.label || '').toLowerCase();
+      const labelB = (b.label || '').toLowerCase();
+
+      if (labelA < labelB) return -1;
+      if (labelA > labelB) return 1;
+      return 0;
+    });
```

Comment:
Since the page is paginated I would do the sorting on the server.

`mu-cl-resources` has a [`:no-case:`](https://github.com/mu-semtech/mu-cl-resources?tab=readme-ov-file#sorting) option which does what we want I think.

The easiest way to set that is to configure the sortable component like this:

`@field=":no-case:label"`

And we also need to update the default value for the `sort` property in the controller.